models/GroupUserDetails.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+    const GroupUserDetails = sequelize.define('GroupUserDetails', {
+
+
+        name: {
+            type: DataTypes.STRING,
+            allowNull: false,
+            unique: true,
+        },
+        description: {
+            type: DataTypes.STRING,
+            unique: false,
```

Comment:
We can remove `unique` here since false is the default.

Let's also set this to `allowNull: false` and `defaultValue: ''`. I've read on the Internetz that allowing null when null is not needed decreases performance of the database. I don't know by how much, but might as well make it a habit.